doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,37 +57,45 @@ Values are provided in kilobytes (KB).
+      | :ref:`Template <matter_template_sample>` (Debug)                   |            28 |               710 |              4 |         32 |         774 |                             174 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Template <matter_template_sample>` (Lto debug)               |            28 |               653 |              4 |         32 |         717 |                             178 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Template <matter_template_sample>` (Release)                 |            28 |               615 |              4 |         32 |         679 |                             168 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Thermostat <matter_thermostat_sample>` (Debug)               |            28 |               761 |              4 |         32 |         825 |                             175 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Thermostat <matter_thermostat_sample>` (Lto debug)           |            28 |               696 |              4 |         32 |         760 |                             179 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Thermostat <matter_thermostat_sample>` (Release)             |            28 |               652 |              4 |         32 |         716 |                             169 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Window Covering <matter_window_covering_sample>` (Debug)     |            28 |               742 |              4 |         32 |         806 |                             175 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Window Covering <matter_window_covering_sample>` (Lto debug) |            28 |               680 |              4 |         32 |         744 |                             178 |
```

Comment:
```suggestion
      | :ref:`Window Covering <matter_window_covering_sample>` (LTO, Debug) |            28 |               680 |              4 |         32 |         744 |                             178 |
```

`Debug` and `Release` are established terms, and `LTO` is just an additional build feature